doc/api/stream.md

```diff
@@ -456,6 +456,16 @@ further errors except from `_destroy()` may be emitted as `'error'`.
 Implementors should not override this method,
 but instead implement [`writable._destroy()`][writable-_destroy].
 
+##### `writable.closed`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+Is `true` after `'close'` has been emitted.
```

Comment:
Nit: Is it also `true` while it is being emitted? Personally, I'd interpret "after `'close`' has been emitted" as in the following snippet, but that's probably not what is meant.

```js
let value = false;
const e = new (require('events').EventEmitter)();

e.addListener('close', () => console.log(value)); // prints false

e.emit('close');

// *after* close has been emitted
value = true;
```

Reply:
true enough, but that's how the other props that work similarly are also documented.